package.json

```diff
@@ -116,7 +117,7 @@
     }
   },
   "peerDependencies": {
-    "carbon-components": "^9.21.0",
+    "carbon-components": "^9.46.0",
```

Comment:
Could we keep the peer dependency as the old one, but update the devDependency? I don't think we want to update peer quite yet for experimental work (unless others feel strongly about this).